test/process_test.cc

```diff
@@ -22,6 +23,37 @@ using namespace testing;
 namespace Nighthawk {
 namespace Client {
 
+constexpr absl::string_view kSinkName = "{name:\"nighthawk.dummy_stats_sink\"}";
+int kNumFlushes = 0;
+
+class DummyStatsSink : public Envoy::Stats::Sink {
+public:
+  DummyStatsSink() : dummy_(1) { kNumFlushes = 0; }
+
+  // Envoy::Stats::Sink
+  void flush(Envoy::Stats::MetricSnapshot&) override { kNumFlushes++; }
+
+  void onHistogramComplete(const Envoy::Stats::Histogram&, uint64_t) override {}
+
+  int dummy_ = 0;
```

Comment:
`private:`?

Reply:
Also, it's not obvious to the reader what `dummy_` is for, maybe add a short comment?

Reply:
The `dummy_ ` has no meaning and is not used, so I removed it.